make-compatible.sh

```diff
@@ -63,15 +63,15 @@ library() {
     done
 }
 
-remove_openstack() {
+remove_rolename() {
     pushd ${ROLE_NAME}/molecule
     # remove the docker scenario files
-    rm -rf openstack
+    rm -rf $ROLENAME
     # remove dependency on oasis_roles.molecule_docker_ci
     sed -i '2d' shared/requirements.yml
     # change any usages of docker to docker
     ls ./docker/molecule.yml
-    grep -lRZ 'openstack' . | xargs -0 -l sed -i -e 's/openstack/docker/g'
+    grep -lRZ '$ROLE_NAME' . | xargs -0 -l sed -i -e 's/$ROLE_NAME/docker/g'
```

Comment:
This actually needs to remain openstack.  By default, there's an openstack scenario and a docker scenario.  This needs to get rid of the openstack scenario since we don't use it